src/vivarium_nih_us_cvd/data/loader.py

```diff
@@ -310,3 +323,53 @@ def load_emr_ihd(key: str, location: str) -> pd.DataFrame:
         data_keys.MYOCARDIAL_INFARCTION.EMR_POST: 15755,
     }
     return _load_em_from_meid(location, map[key], "Excess mortality rate")
+
+
+def modify_rr_affected_entity(
+    data: pd.DataFrame, risk_key: str, mod_map: Dict[str, List[str]]
```

Comment:
Nit, and not worth changing your PR, but FWIW, you may want to end these lists with a `,` so black formats the parameters on each line like in your `handle_special_cases` method.

Reply:
Oh is THAT why black sometimes breaks it up and sometimes not? I kinda just assumed it was the 80 or 100 character column limit